pwndbg/lib/arch.py

```diff
@@ -39,4 +36,4 @@ def signed(self, integer: int) -> int:
         return self.unpack(self.pack(integer), signed=True)  # type: ignore
 
     def unsigned(self, integer: int) -> int:
-        return self.unpack(self.pack(integer))
+        return self.unpack(self.pack(integer))  # type: ignore
```

Comment:
Can you explain why we need the `type: ignore`? I like to actually put the error message we're ignoring in a comment so it's clear why we need the `type: ignore`.

Reply:
It is caused by a wrong type in `unpack`, already fixed.